### PR TITLE
Logic, tests and docs for deletion of experiments from scxa tables.

### PR DESCRIPTION
**Pull request by pcm32**
This PR adds logic, tests and docs for deletion of experiments from scxa tables.

### CODE REVIEW

File: tests/random-data-set.bats [hunk 1]
```diff
@@ -101,6 +113,27 @@
...
+}
+
+@test "Marker genes: Delete rows for experiment" {
+  countBefore=$(echo "SELECT COUNT(*) FROM scxa_marker_genes" | psql $dbConnection | awk 'NR==3')
+  export EXP_ID=TEST-EXP2
+  run delete_db_scxa_marker_genes.sh
+  echo "output = ${output}"
+  [ "$status" -eq 0 ]
+  countAfter=$(echo "SELECT COUNT(*) FROM scxa_marker_genes" | psql $dbConnection | awk 'NR==3')
+  [ $(( countBefore - countAfter )) == 274 ]
```
Comment (suhaibMo) [inline tests/random-data-set.bats:134]: Is 274 difference in counts for marker genes is expected for all experiments ?

Comment (pcm32) [reply tests/random-data-set.bats:134]: It is for the two example experiments (the same but just duplicated with different accessions) that are loaded in the tests (which is controlled within the repo).

**Review by suhaibMo — APPROVED** [review]
Overall seems good !

**Comment by pcm32** [comment]
Thanks @suhaibMo

**Pull request merged**